catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswRecordConverter.java

```diff
@@ -224,6 +230,32 @@ public Metacard transform(InputStream inputStream, String id)
     public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
             throws CatalogTransformerException {
         StringWriter stringWriter = new StringWriter();
+        if (StringUtils.isNotBlank(metacard.getMetadata())) {
+            // Check if the metadata is csw:Record
+            try {
+                StringReader xml = new StringReader(metacard.getMetadata());
+                XMLEventReader reader = factory.createXMLEventReader(xml);
+                boolean rootFound = false;
+                while (reader.hasNext() && !rootFound) {
+                    XMLEvent event = reader.nextEvent();
+                    if (event.isStartElement()) {
+                        rootFound = true;
+                        StartElement startElement = event.asStartElement();
```

Comment:
looks like if you were to call this as event.asStartElement().getName() you could shorten the calls later on